data_ingestion/012_Ingest_text_data_v2.ipynb

```diff
@@ -0,0 +1,1034 @@
+{
```

Comment:
<p>Run on sentence - please split up. </p><blockquote>Labeled text data sometimes are in structured data format, such as reviews for sentiment analysis, news headlines for topic modeling, or documents for text classification, where you have one column for the label, one column for the text, and sometimes other columns as attributes, and you can just treat them as tabular data and ingest them as we talked about in last section. </blockquote><p>I.e.</p><blockquote>Labeled text data can be in a structured data format, such as reviews for sentiment analysis, news headlines for topic modeling, or documents for text classification. In these cases, you may have one column for the label, one column for the text, and sometimes other columns for attributes. You can treat this structured data like tabular data, and ingest it in one of the ways discussed in the previous notebook (011_Ingest_tabular_data_v1.ipynb)[ADD LINK TO NOTEBOOK]. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>